projects/power_distribution/src/front_uv_detector.c

```diff
@@ -0,0 +1,30 @@
+#include "front_uv_detector.h"
+
+#include "can_transmit.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "pin_defs.h"
+
+static GpioAddress s_uv_comp_pin_address = FRONT_UV_COMPARATOR_PIN;
```

Comment:
This variable is unused now, please remove it. (And you can also remove the `pin_defs.h` include.)